Firebase/Storage/FIRStorageDeleteTask.m

```diff
@@ -21,6 +21,13 @@ @implementation FIRStorageDeleteTask {
   FIRStorageVoidError _completion;
 }
 
+@synthesize fetcher = _fetcher;
+@synthesize fetcherCompletion = _fetcherCompletion;
+
+- (void) dealloc {
+  [_fetcher stopFetching];
+}
```

Comment:
add `[super dealloc]` as the last line to ensure that deallocation continues appropriately